internal/controller/lease_controller.go

```diff
@@ -465,15 +465,13 @@ func filterOutOfflineExporters(matchingExporters []jumpstarterdevv1alpha1.Export
 	onlineExporters := slices.DeleteFunc(
 		matchingExporters,
 		func(exporter jumpstarterdevv1alpha1.Exporter) bool {
-			return !(true &&
```

Comment:
Those two expressions are equivalent, why the change :)

I had to refresh my De Morgan's neurons X-DD

Reply:
golangci-lint thinks this one is better!